source/how-tos/debug/debug-apache.rst

```diff
@@ -77,7 +79,50 @@ Or you're using the wrong hostname in your browser.
 
           sudo /sbin/apache2 -S
 
+Performance Tuning
+------------------
+
+If you're servicing many clients at time (more than 50) you will likely need to change the
+`Apache Httpd's MPM configuration`_. The default configuration may degrade service when
+Httpd has to serve many clients (I.e., when you have a lot of customers using Open OnDemand).
+
+We suggest configurations similar to this. 
+
+.. note::
+  The most important directives used to control this MPM are `ThreadsPerChild`, which controls 
+  the number of threads deployed by each child process and `MaxRequestWorkers`, which controls
+  the maximum total number of threads that may be launched.
+  
+  MaxRequestWorkers * ThreadsPerChild will be your effictive cap on the number of 
+  simultaneous requests that will be served.
+   
+
+.. code-block:: apache
+
+  # conf.modules.d will vary depending on the platform and version.
+  # $APACHE_HOME/conf.modules.d/mpm.conf
+  
+  # MPM event is actually important for idle VNC connections alive.
+  LoadModule mpm_event_module modules/mod_mpm_event.so
```

Comment:
This file still has the `Load` statement which I'm sure is not the best practice, but IDK how else to convey that you also need to do this (and different platforms have different schemes).

Reply:
`LoadModule` is necessary to load this MPM but it's not always in same file as the config but doesn't hurt to have them in same file.  It's also important people remove referenecs to `mod_mpm_prefork.so` which is the default.  So by default there is a line like this:

```
LoadModule mpm_prefork_module modules/mod_mpm_prefork.so
```
That line must be deleted and replaced by one to load event MPM.  Can't have both loaded at same time, pretty sure Apache will refuse to start if you do.

Reply:
Updated the comment, it appears to be commented out in a default install.